inferentia/scripts/gen_triton_model.py

```diff
@@ -337,32 +526,50 @@ def create_model_file():
         type=int,
         default=0,
         help='The maximum batch size for the model being generated')
+    parser.add_argument('--is_tensorflow_model',
```

Comment:
Why are you using a flag specific to tf? Just use `--model_type` and default it to "torchscript". Allow either torchscript or "savedmodel". This would be more expressive in my opinion and allow you to easily extend to support more model type in the future.